package.json

```diff
@@ -61,11 +59,8 @@
     "webpack-cli": "3.3.12"
   },
   "lint-staged": {
-    "*.{js}": [
-      "eslint --ignore-path .gitignore --ignore-path .prettierignore --fix"
-    ],
```

Comment:
I think we should keep ESLint enabled. It's useful to run on change files as the user may not have an editor integration, especially now that `yarn test` isn't running ESLint.

Reply:
I'm personally really not a fan of linting on commit, since I commit often and and tend to amend commits. I think it's a good point about not running the linter for `yarn test`, though. Maybe can add the linting script back to `test` and then have a separate script for CI?

Reply:
Optionally we can run lint/tests before pushing to remote by adding it to pre-push step using husky.

Reply:
Addressed these comments. Please re-review.